Core/include/Acts/Seeding/SeedfinderConfig.hpp

```diff
@@ -150,6 +151,16 @@ struct SeedfinderConfig {
   int nTrplPerSpBLimit = 100;
   int nAvgTrplPerSpBLimit = 2;
 
+  // Delegates for accessors to detailed information on double measurement that
+  // produced the space point
+  bool useDetailedDoubleMeasurementInfo = false;
```

Comment:
In case we want something similar for pixel clusters, we would need another set of Delegates, which I guess is unavoidable.. unless you have a good idea how to avoid cluttering the config even more?
Do you think this naming `useDetailedDoubleMeasurementInfo` is clearly transporting "stereo angle strip details" to someone from a different experiment?
You are definitely the expert in this matter, so if you say "that's how it should look" then that's a fact ;)

Reply:
To get it more general, I am going to replace `strip` with `measurement` and make a clear comment about the meaning of the quantities. 
To give as well an explanation of the use of these quantities, they allow to implement a compatibility check between the position of the candidate space points (bottom-middle-top, hence a seed direction) and the channels (in this particular case strips) used to build the space points. If the seed direction is compatible with the channels fired, the candidate space points move to the next checks of seed production. If, instead, the seed direction requires the space points to "move" outside the channel length, i.e. outside the detector element, the combination of the three space points is discarded, reducing the quantity of low quality seeds.